infra/analytics/app-config/env-config/environment-variables.tf

```diff
@@ -61,6 +61,8 @@ locals {
     #     -- the "app" database is where the analytics data is stored
     #     GRANT ALL PRIVILEGES ON DATABASE app TO metabaseuser;
     #     GRANT CONNECT ON DATABASE app TO metabaseuser;
+    #     GRANT USAGE ON SCHEMA app TO metabaseuser;
+    #     GRANT SELECT, INSERT, UPDATE, DELETE ON ALL TABLES IN SCHEMA app TO metabaseuser;
```

Comment:
These have to be run after the schema is created, and it hasn't been created yet in staging / prod, so this only works in dev so far.

Reply:
Also, dunno if I should scope this down to just `SELECT`... I probably should?

Reply:
Metabase suggests setting the following permissions:

https://www.metabase.com/docs/latest/databases/users-roles-privileges#minimum-database-privileges

Reply:
So yeah, looks like `SELECT` should be fine for now!

Reply:
Ahhh! I had no idea they wrote a guide